app/assets/stylesheets/application.scss

```diff
@@ -21,20 +21,25 @@ $govuk-global-styles: true;
 }
 
 .task-checkboxes {
-  .govuk-checkboxes__item {
-    margin-bottom: govuk-spacing(9);
-  }
-}
-
-.task-checkboxes,
-.clear-legal-documents-task-checkboxes {
   .govuk-checkboxes__item {
     label {
       @extend .govuk-\!-font-weight-bold;
     }
   }
 }
 
+.task-action__wrapper:first-of-type {
```

Comment:
Is ti worth scoping these within the `task-checkboxes` class (or other similar container class) to keep things organised?